### PR TITLE
docs: updated create-medusa-app based on admin plugin changes

### PR DESCRIPTION
**Pull request by shahednasser**
Updated `create-medusa-app` to add notes about how the admin plugin is now installed in the mix.

Closes DOCS-582

### CODE REVIEW

File: docs/content/create-medusa-app.mdx [hunk 1]
```diff
@@ -13,17 +13,15 @@ In this document, you’ll learn how to use `create-medusa-app` to create a Medu
 
 ## Overview
 
-Medusa is composed of three different components: the headless backend, the storefront, and the admin dashboard.
+Medusa is composed of three different components: a Node.js backend, the storefront, and the admin dashboard.
```
Comment (olivermrbl) [inline docs/content/create-medusa-app.mdx:16]: suggestion: We should rewrite this overview section to align with our updated positioning. What I mean is Medusa is no longer only composed of three different components, but we are now moving toward being a toolkit for any type of commerce application. 

I can write out a suggestion, if you think it makes sense.

Comment (shahednasser) [reply docs/content/create-medusa-app.mdx]: Thanks for the suggestion! I think that makes a lot of sense. I changed up the overview section to reflect that, but feel free to make suggestions for additional changes in it

File: docs/content/create-medusa-app.mdx [hunk 2]
```diff
@@ -13,17 +13,15 @@ In this document, you’ll learn how to use `create-medusa-app` to create a Medu
```
Comment (olivermrbl) [inline docs/content/create-medusa-app.mdx:13]: todo: The introduction in the doc should also reflect the updated message:
```
In this document, you’ll learn how to use create-medusa-app to set up a Medusa server and an optional storefront.
```

File: docs/content/create-medusa-app.mdx [hunk 3]
```diff
@@ -13,17 +13,15 @@ In this document, you’ll learn how to use `create-medusa-app` to create a Medu
 
 ## Overview
 
-Medusa is composed of three different components: the headless backend, the storefront, and the admin dashboard.
+Medusa provides developers with the toolkit to create a commerce application including a Node.js backend that encapsulates the core package and the commerce modules, an admin dashboard, and a storefront. Developers can set up and use these tools separately, giving them a full ownership over their tech stack.
 
-Medusa provides the necessary tools and resources to set up the three components separately. This ensures that developers have full freedom to choose their tech stack, as they can choose any framework for the storefront and admin dashboard.
+In certain cases, it can make more sense to setup all the main building blocks together, such as when you're creating a fully fledged ecommerce store, or if you're curious about how Medusa works and what features it offers. For these cases, you can use the `create-medusa-app` command.
```
Comment (olivermrbl) [inline docs/content/create-medusa-app.mdx:18]: todo: I think we can leave out this paragraph, as you can use `create-medusa-app` regardless of how you want to build your applications.

File: docs/content/create-medusa-app.mdx [hunk 4]
```diff
@@ -13,17 +13,15 @@ In this document, you’ll learn how to use `create-medusa-app` to create a Medu
...
 
-However, if you’re interested in using Medusa’s starters for the three components, you can make use of the `create-medusa-app` command instead of creating each separately.
-
-When you run the `create-medusa-app` command, you’ll install a Medusa backend, a Medusa admin, and optionally a storefront at the same time.
+When you run the `create-medusa-app` command, you’ll install a Medusa backend, and optionally a Medusa admin and a storefront at the same time.
```
Comment (olivermrbl) [inline docs/content/create-medusa-app.mdx:20]: todo: We can remove this, as this is already mentioned in the beginning of the overview.
```suggestion
```

File: docs/content/create-medusa-app.mdx [hunk 5]
```diff
@@ -13,17 +13,15 @@ In this document, you’ll learn how to use `create-medusa-app` to create a Medu
...
 
 :::note
 
-If you instead want to quickly install and setup only a Medusa backend, follow [this quickstart guide](./development/backend/install.mdx).
+If you instead want to quickly install and setup only a Medusa backend, follow [this quickstart guide](./development/backend/install.mdx) instead.
```
Comment (olivermrbl) [inline docs/content/create-medusa-app.mdx:24]:
```suggestion
If you only want to set up a Medusa server, follow [this quickstart guide](./development/backend/install.mdx) instead.
```

File: docs/content/create-medusa-app.mdx [hunk 6]
```diff
@@ -13,17 +13,15 @@ In this document, you’ll learn how to use `create-medusa-app` to create a Medu
 
 ## Overview
 
-Medusa is composed of three different components: the headless backend, the storefront, and the admin dashboard.
+Medusa provides developers with the toolkit to create a commerce application including a Node.js backend that encapsulates the core package and the commerce modules, an admin dashboard, and a storefront. Developers can set up and use these tools separately, giving them a full ownership over their tech stack.
```
Comment (olivermrbl) [inline docs/content/create-medusa-app.mdx:16]:
```suggestion
Medusa is a toolkit for developers to create digital commerce applications. In its simplest form, Medusa is a Node.js server with the core API, plugins, and modules installed through npm. In this Node.js server, you can extend your application with custom entities, endpoints, services, and more. You can learn more about customizing Medusa in the dedicated documentation.
```

**Review by olivermrbl — CHANGES_REQUESTED** [review]
This is very nice! Added a couple of suggestions and todos :)

**Comment by shahednasser** [comment]
Thanks for the comments @olivermrbl ! I think you provide great points, but at the same time the overview section is becoming something not really related to the `create-medusa-app` in particular. I recommend either:

1. Add a sentence of how the first paragraph "Medusa is a toolkit..." ties into the `create-medusa-app` in particular and why it's important to know that with relation to the current guide.
2. or remove the overview in its entirety and jump right into the prerequisites and steps. We can also add a link to the core concepts overview page which can help the developer learn more about Medusa's architecture if necessary.

What do you think? I recommend the first approach which gives the developers an overall idea of what Medusa is while still informing them of how it ties into the guide they're about to follow, but I'm fine with both approaches.

**Comment by olivermrbl** [comment]
> What do you think? I recommend the first approach which gives the developers an overall idea of what Medusa is while still informing them of how it ties into the guide they're about to follow, but I'm fine with both approaches.

I see your point - I probably took it a couple of steps too far 😅 

I like the first approach. My suggestion for the paragraph would be:
```
Medusa is a toolkit for developers to create digital commerce applications. In its simplest form, Medusa is a Node.js server with the core API, plugins, and modules installed through npm.
```

Also, I'll update my suggestions to reflect your comment, and you can let me know what you think about that 💯

**Comment by olivermrbl** [comment]
> What do you think? I recommend the first approach which gives the developers an overall idea of what Medusa is while still informing them of how it ties into the guide they're about to follow, but I'm fine with both approaches.

I see your point - I probably took it a couple of steps too far 😅 

I like the first approach. My suggestion for the paragraph would be:
```
Medusa is a toolkit for developers to create digital commerce applications. In its simplest form, Medusa is a Node.js server with the core API, plugins, and modules installed through npm.
```

Also, I'll update my suggestions to reflect your comment, and you can let me know what you think about that 💯

**Comment by olivermrbl** [comment]
> What do you think? I recommend the first approach which gives the developers an overall idea of what Medusa is while still informing them of how it ties into the guide they're about to follow, but I'm fine with both approaches.

I see your point - I probably took it a couple of steps too far 😅 

I like the first approach. My suggestion for the paragraph would be:
```
Medusa is a toolkit for developers to create digital commerce applications. In its simplest form, Medusa is a Node.js server with the core API, plugins, and modules installed through npm.
```

Also, I'll update my suggestions to reflect your comment, and you can let me know what you think about that 💯

**Comment by olivermrbl** [comment]
> What do you think? I recommend the first approach which gives the developers an overall idea of what Medusa is while still informing them of how it ties into the guide they're about to follow, but I'm fine with both approaches.

I see your point - I probably took it a couple of steps too far 😅 

I like the first approach. My suggestion for the paragraph would be:
```
Medusa is a toolkit for developers to create digital commerce applications. In its simplest form, Medusa is a Node.js server with the core API, plugins, and modules installed through npm.
```

Also, I'll update my suggestions to reflect your comment, and you can let me know what you think about that 💯

**Review by olivermrbl — APPROVED** [review]
LGTM - great one!

**Pull request merged**